### PR TITLE
06 sep

### PR DESCRIPTION
**Pull request by Kapil0187**

### CODE REVIEW

File: app/models/flight.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Flight < ApplicationRecord
+  validates :name, :total_seets, :departure_time, :arrival_time, :price, :direction_id, presence: true
+
```
Comment (Sahilchouhan20) [inline app/models/flight.rb:3]: remove the extra line below validates.

File: app/views/allbookings/history.html.erb [hunk 2]
```diff
@@ -18,7 +18,6 @@
           <th scope="col"> Date </th>
           <th scope="col"> Type </th>
           <th scope="col"> Tickets </th>
-          <th scope="col"> Cancel Bookings </th>  
         </tr>
       </thead>
       <tbody>
```
Comment (Sahilchouhan20) [inline app/views/allbookings/history.html.erb:23]: follow indentation

File: spec/models/direction_spec.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Direction, type: :model do
+
```
Comment (Sahilchouhan20) [inline spec/models/direction_spec.rb:4]: why you use extra line here.

Comment (Kapil0187) [reply spec/models/direction_spec.rb]: okay i will remove extra line

File: app/views/allbookings/history.html.erb
Comment (ShahirMansuri786) [inline app/views/allbookings/history.html.erb:1]: please use one blank line after the end file

File: spec/models/bus_spec.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  it 'is not valid without a bustype' do
+    bus.bustype = nil
+    expect(bus).to_not be_valid
+  end
+
+  it 'is not valid without a remaning_seats' do
+    bus.remaning_seats = nil
+    expect(bus).to_not be_valid
+  end
+end
```
Comment (ShahirMansuri786) [inline spec/models/bus_spec.rb:19]: please use one blank line after the end of the file

**Pull request merged**